vm_scripts/take_snapshot.py

```
@@ -0,0 +1,347 @@
+
+        # init helpers
+        vm_helper = VmHelper()
+
+        # Get the VM object
+        vm_obj_response = vm_helper.get_vm_obj(vm_name, defaults['data_center'], defaults['host'], str_resource_pool, str_folder, defaults['is_cluster'])
+
+
+        if vm_obj_response and vm_obj_response["status"] == "success" and vm_obj_response["res"] is not None:
+            vm = vm_obj_response["res"]
+        else:
+            logger_message = "Error while taking snapshot for VM: ()".format(str(vm_obj_response["res"]))
```

Ruff (0.8.2)

`.format` call has unused arguments at position(s): 0

Remove extra positional arguments at position(s): 0

(F523)